release_notes/ocp-4-3-release-notes.adoc

```diff
@@ -722,8 +759,11 @@ Service Mesh from version 1.0.1 to 1.0.2].
 * The `ImageContentSourcePolicy` is not supported by the registry at the time of
 release. In disconnected environments, Jenkins can be enabled to pull through by
 default. Use this command as a workaround to use Jenkins in disconnected
-environments: 
+environments:
 +
-====
+----
 oc tag <jenkins_source_image> jenkins:2 --reference-policy=source -n openshift
-====
+----
++
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1787112[*BZ#1787112*])
```

Comment:
I don't know if our format for this is immutable, but it looks odd having the BZ link dangling after the code block. I wonder if it's permissible to move it above the code block?

Reply:
Yeah, I was struggling with this. I did see it done in the 4.2 release notes, which is why I moved forward with it. But I'm totally in agreeance that it looks kind of funky. I'll move it above the code block. :+1: